packages/block-library/src/paragraph/edit.js

```diff
@@ -52,6 +58,115 @@ function hasDropCapDisabled( align ) {
 	return align === ( isRTL() ? 'left' : 'right' ) || align === 'center';
 }
 
+function ParagrapgShortcuts( { attributes, clientId } ) {
+	const { align, content } = attributes;
+	const { replaceBlocks } = useDispatch( blockEditorStore );
+	const { registerShortcut } = useDispatch( keyboardShortcutsStore );
+	registerShortcut( {
+		name: 'core/blocks/paragraph/transform-to-heading-1',
+		category: 'block-library',
+		description: __( 'Transform to heading.' ),
+		keyCombination: {
+			modifier: 'access',
+			character: '1',
+		},
```

Comment:
On Windows, Ctrl+Alt+{number} keys are used to open the pinned apps in the corresponding position one the taskbar, so I don't think this is a good shortcut combination.

Reply:
That is what @ellatrix suggested. On OSX this works swell. @t-hamano you've done some windows compat fixes, do you have any suggestion as to what could work without conflicts on Wirdows?

Reply:
In Windows, I think the shortcut to open a pinned app is "Windows logo key + number" (Please see "Windows logo key keyboard shortcuts" section).
https://support.microsoft.com/en-us/windows/keyboard-shortcuts-in-windows-dcc61a57-8ff0-cffe-9796-cb9706c75eec

Also, this is very complicated, but when the modifier is `access`, it means `CTRL + ALT` on Mac and `SHIFT + ALT` on Windows.
https://github.com/WordPress/gutenberg/blob/b66257558a7c7d3bffd88c5c9d3e4e75a0c7e828/packages/keycodes/src/index.js#L164

I report that the current implementation works as expected on Windows 🙂

<details>
<summary>Screencast on Windows</summary>

https://user-images.githubusercontent.com/54422211/194326122-d5571f80-7867-487a-9c2a-c816f2adecde.mp4

</details>

Reply:
> In Windows, I think the shortcut to open a pinned app is "Windows logo key + number" (Please see "Windows logo key keyboard shortcuts" section).

You're right, I forgot that my keyboard has alt and windows keys swapped over because I usually work on a mac 😆